vpn/view.js

```diff
@@ -1,36 +0,0 @@
-'use strict';
```

Comment:
don't remove this, it is extremely useful to query UPNP using this tool.

Reply:
reverted